pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -258,17 +258,17 @@ void ATA::CreateHDDinfo(u64 sizeSectors)
 		(1 << 12) |							//Fixed
 		(1 << 13)));						//Fixed      //word 86
 	//Command set/feature enabled/supported (See word 84)
-	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 1) | 1));
-	WriteUInt16(identifyData, &index, static_cast<u16>(
-		(1) |								//Fixed
-		((1) << 1)));						//Fixed      //word 87
+	WriteUInt16(identifyData, &index, static_cast<u16>((1 << 14) | (1 << 1) | 1));      //word 87
+	// WriteUInt16(identifyData, &index, static_cast<u16>(
+	//	(1) |								//Fixed
+	//	((1) << 1)));						//Fixed
```

Comment:
```suggestion
	WriteUInt16(identifyData, &index, static_cast<u16>(
	(1) |								//Fixed
	(1 << 1) |							//Fixed
	(1 << 14)));						//Fixed      //word 87
```

Reply:
nope, its not an error. Word 87 is written twice thus leads to shifting of UDMA control